raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -579,74 +585,112 @@ contract TokenNetwork is Utils {
 
     }*/
 
+    function updateBalanceProofData(
```

Comment:
We are updating `balance_hash` and `nonce` =  balance proof data in my opinion

`updateBalanceHashData` - confusing because we have `balance_hash`
`updateBalanceProofHashData` - technically, possibly more correct because signing does hashing before, but I still think `updateBalanceProofData` is ok and shorter. And again, I don't want confusion with `balance_hash`

If I say `updateBalanceProofData` - what does it mean to you?

Reply:
> If I say updateBalanceProofData - what does it mean to you?

I think about the whole balance proof transferred amount, merkleroot, nonce, etc.

> And again, I don't want confusion with balance_hash

It's exactly because the function is updating the balance_hash that I gave that suggestion (and because I would call it `balanceproof_hash` :P ):

```solidity
balance_state.balance_hash_or_locksroot = balance_hash;
```